docker/backup.sh

```diff
@@ -6,5 +6,11 @@ backup_dir="backups/$(date +%FT%H%M%S)"
 
 mkdir -p "${backup_dir}"
 
-docker compose exec core     tar czv /app/data      > "${backup_dir}/core_data.tar.gz"
+TMP_CORE_NAME=$(docker compose ps -f  '{{.Names}}' | grep core || echo "Error: No running services found.")
+export TMP_CORE_NAME
+echo "Name of Core's container is: $TMP_CORE_NAME"
+
+docker exec $TMP_CORE_NAME   tar czv /app/data      > "${backup_dir}/core_data.tar.gz"
 docker compose exec database pg_dumpall -U taranis  > "${backup_dir}/database_backup.sql"
```

Comment:
if this line works as expected why doesn't the one above?

Reply:
After further inspection, it seems that the problem could come from 
1. the `tar` command as well, as it may be redirecting different data to the other one.
2. maybe the container itself has some constraints

However, tt actually copies the file to host, just generates an error, so the script stops. If we let the database backup first and core second, it copies over the two backups. 
Ends with the same error 
`Error: executing /usr/bin/podman-compose exec core tar czv - /app/data: exit status 2`

Reply:
the tar file is actually corrupted with content:
```$ cat core_data.tar.gz 
tar: Refusing to write archive contents to terminal (missing -f option?)
tar: Error is not recoverable: exiting now```

Reply:
I implemented a workaround using a file. Couldn't find any other solution.